File: ReelMarkets/Assets/Scripts/LoginSession.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;
using UserItems;

namespace Session
{

    public static class LoginSession
    {


        public static User currentUser { get; set; }

        public static bool isValidSession()
        {
            if (currentUser != null)
            {
                return true;
            }
            return false;
        }

        public static void clearSession()
        {
            currentUser = null;
        }

    }
}

File: app/src/main/java/com/cmdev/reelmarkets/model/LoginSession.java
package com.cmdev.reelmarkets.model;

/**
 * Created by Vagdevi on 2/1/2017.
 */

public class LoginSession {
    public static User currentUser;

    public static User getCurrentUser() {
        return currentUser;
    }

    public static void setCurrentUser(User currentUser) {
        LoginSession.currentUser = currentUser;
    }

    public static boolean isValidSession()
    {
        if (currentUser != null)
        {
            return true;
        }
        return false;
    }

    public static void clearSession()
    {
        currentUser = null;
    }
}
